janus-idp/backstage-plugins
# fix(tekton): fix incorrect image path for signed badge svg

**Fixes:**

https://github.com/janus-idp/backstage-plugins/issues/2099

**Description:**

Latest Backstage build is not able to resolve image files (signed-badge.svg), so converted the svg into react component.

**Screenshots:**

![image](https://github.com/user-attachments/assets/5729adac-70b8-4af7-bbc6-9470a97b7125)

**How to test:** 
1. Navigate into the `plugins/tekton` folder and run `yarn start` to start the application in dev mode.
2. Signed badge will appear on the last pipelinerun in that list.
